hypertrace-bom/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+  `java-platform`
+  id("org.hypertrace.publish-plugin")
+}
+
+javaPlatform {
+  allowDependencies()
+}
+
+dependencies {
+  api(platform(libs.grpc.bom))
```

Comment:
other dependency platforms/boms provide constraints too - so adding them as direct basically merges their constraints with the other ones declared here. If they were added as constraints, it would instead constrain the version of the bom they use (e.g. a project that declared grpc-bom would get the right version, but a project the declared grpc-api would not). Hope that makes sense.

Reply:
Got it. Thanks.